cluster-provision/k8s/1.18/provision.sh

```diff
@@ -230,7 +230,7 @@ rules:
 EOF
 
 cat > /etc/kubernetes/kubeadm.conf <<EOF
-apiVersion: kubeadm.k8s.io/v1beta1
+apiVersion: kubeadm.k8s.io/v1beta2
```

Comment:
THe commit message says "remove fields that dont have an impact.". I don't see any field removed in this patch.

Reply:
https://github.com/kubevirt/kubevirtci/pull/427/commits/5977795be803c63010884967861dc4c3a20ec8c9#diff-6471a431c9f4f9344280f4fefe27462bL264

Reply:
How did I miss it...